gsy_framework/forward_markets/aggregated_ssp_profile.py

```diff
@@ -0,0 +1,165 @@
+import csv
+from abc import ABC, abstractmethod
+from pathlib import Path
+from typing import Dict, Iterable
+
+from pendulum import DateTime, duration, period
+
+from gsy_framework.enums import AggregationResolution
+from gsy_framework.forward_markets.forward_profile import (
+    ProfileScaler, StandardProfileParser, gsy_framework_path)
+
+RESOURCES_PATH = Path(gsy_framework_path) / "resources"
+
+
+class AggregatedSSPProfileBase(ABC):
+    """Base class representing the Standard Solar Profile in different resolutions."""
+
+    SSP_AGGREGATED_AGGREGATED_PROFILE_PATH: Path = None
+    LEAP_YEAR_SSP_AGGREGATED_PROFILE_PATH: Path = None
+
+    def __init__(self, capacity_kWh: float):
+        self.capacity_kWh = capacity_kWh
```

Comment:
There seem to be a few pylint warnings in this file. Could you check and maybe disable the unnecessary ones?